CONTRIBUTING.md

```diff
@@ -28,6 +28,7 @@ To run linting:
 
 ```bash
 flake8 .
```

Comment:
https://github.com/wemake-services/dotenv-linter/ has `flake8`, but I am not 100% convinced we actually still need it? Let me know and I can remove it.